doc/examples/reconst_bingham.py

```diff
@@ -0,0 +1,304 @@
+# .. rst-class:: centered small fst-italic fw-semibold
+#
+# ODFs reconstructed using the constrained Spherical Deconvolution (CSD).
+#
+#
+# Step 2. Bingham fitting and Metrics
+# =================================================
+# Now that we have some ODFs, let's fit the Bingham functions to them by using
+# the function `bingham_from_odf`:
+
+BinghamMetrics = bingham_from_odf(csd_odf, sphere)
+
+###############################################################################
+# The above function outputs a `BinghamMetrics` class instance, containing the
+# parameters of the fitted Bingham functions. For instance, the fitted Bingham
+# functions can be visualized using the following lines of code:
+
+bim_odf = BinghamMetrics.odf(sphere)
+
+scene.rm(fodf_spheres)
+
+fodf_spheres = actor.odf_slicer(bim_odf, sphere=sphere, scale=0.9,
+                                norm=False, colormap='plasma')
+scene.add(fodf_spheres)
+
+print('Saving illustration as Bingham_odfs.png')
+window.record(scene, out_path='Bingham_odfs.png', size=(600, 600))
+if interactive:
+    window.show(scene)
+
+###############################################################################
+# .. rst-class:: centered small fst-italic fw-semibold
+#
+# Bingham functions fitted to CSD's ODFs.
+#
+# Alternatively to fit Bingham functions to sampled ODFs, DIPY also contains
+# the function `bingham_from_sh` to perform Bingham fitting from the ODF's
+# spherical harmonic representation. Although this process may require longer
+# processing times, this function may be useful to avoid memory issues in
+# handling heavily sampled ODFs. Below we show the lines of code to use
+# function `bingham_from_sh` (feel free to skip these lines if the function
+# `bingham_from_odf` worked fine for you). Note, to use `bingham_from_sh` you
+# need to specify the maximum order of spherical harmonics that you defined in
+# `csd_model` (in this example this was set to 8):
+
+sh_coeff = csd_fit.shm_coeff
+BinghamMetrics = bingham_from_sh(sh_coeff, sphere, 8)
+
+###############################################################################
+# Step 3. Bingham Metrics
+# =================================================
+# As mentioned above, reconstructed Bingham functions can be useful to
+# quantify properties from ODFs [1]_, [2]_. Below we plot the Bingham metrics
+# expected to be proportional to the fiber density (FD) of specific fiber
+# populations.
+
+FD_ODF_l1 = BinghamMetrics.fd[:, :, 0, 0]
+FD_ODF_l2 = BinghamMetrics.fd[:, :, 0, 1]
+FD_total = BinghamMetrics.tfd[:, :, 0]
+
+fig1, ax = plt.subplots(1, 3, figsize=(16, 4))
+
+im0 = ax[0].imshow(FD_ODF_l1[:, -1:1:-1].T, vmin=0, vmax=2)
+ax[0].set_title('FD ODF lobe 1')
+
+im1 = ax[1].imshow(FD_ODF_l2[:, -1:1:-1].T, vmin=0, vmax=2)
+ax[1].set_title('FD ODF lobe 2')
+
+im2 = ax[2].imshow(FD_total[:, -1:1:-1].T, vmin=0, vmax=2)
+ax[2].set_title('FD total')
+
+fig1.colorbar(im0, ax=ax[0])
+fig1.colorbar(im1, ax=ax[1])
+fig1.colorbar(im2, ax=ax[2])
```

Comment:
check in dipy.viz.plotting is there not a function that satisfy your needs. otherwise, create one or update one

Reply:
Created a new `image_mosaic` method in `viz.plotting`.